docs/api/app.md

```diff
@@ -1307,6 +1307,76 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.sysInfo.numberOfProcessors()`
+
+Returns `Number` - The number of logical processors/cores on the current machine.
+
+### `app.sysInfo.amountOfPhysicalMemory()`
+
+Returns `Number` - The number of bytes of physical memory on the current machine.
+
+### `app.sysInfo.amountOfAvailablePhysicalMemory()`
+
+Returns `Number` - The number of bytes of current available physical memory on the machine.
+
+### `app.sysInfo.amountOfVirtualMemory()`
+
+Returns `Number` - The number of bytes of virtual memory of this process.
+
+**NOTE:** A return value of zero means that there is no limit on the available virtual memory.
+
+### `app.sysInfo.amountOfFreeDiskSpace(path)`
+
+* `path` String
+
+Returns `Number` - The available disk space in bytes on the volume containing `path`.
+
+**NOTE:** Returns -1 on failure.
+
+### `app.sysInfo.amountOfTotalDiskSpace(path)`
+
+* `path` String
+
+Returns `Number` - The total disk space in bytes on the volume containing `path`.
+
+**NOTE:** Returns -1 on failure.
+
+### `app.sysInfo.uptime()`
+
+Returns `Number` - The system uptime in seconds.
+
+### `app.sysInfo.getHardwareInfo()`
+
+Returns `Promise<HardwareInfo>` - Resolves with a [HardwareInfo](structures/hardware-info.md)
+
+### `app.sysInfo.operatingSystemName()`
+
+Returns `String` - The name of the host operating system.
+
+### `app.sysInfo.operatingSystemVersion()`
+
+Returns `String` - The version of the host operating system.
+
+### `app.sysInfo.operatingSystemArchitecture()`
+
+Returns `String` - The architecture of the running operating system.
+
+**NOTE:** Exact return value may differ across platforms.
+
+### `app.sysInfo.cpuModelName()`
+
+Returns `String` - The CPU model name of the system.
+
+**NOTE:** If it can not be determined, an empty string is returned.
+
+### `app.sysInfo.vmAllocationGranularity()`
```

Comment:
These seems really low-level, do we need to expose this?

Reply:
removed